Bio/SearchIO/BlastIO/blast_tab.py

```diff
@@ -212,6 +212,7 @@ def __init__(self, handle, comments=False, fields=_DEFAULT_FIELDS):
         self.line = self.handle.readline().strip()
 
     def __iter__(self):
+        """Iterate over BlastTabParser, yield query result."""
```

Comment:
You've not been consistent with yield/yields, and I would expect results plural given the context. How about "..., yields query results." here (and on the other entries)?

Reply:
@peterjc you are right, amended.